code/game/machinery/machinery.dm

```diff
@@ -196,6 +196,18 @@ Class Procs:
 /obj/machinery/proc/update_use_power(var/new_use_power)
 	use_power = new_use_power
 
+// Sets the power_channel var
+/obj/machinery/proc/update_power_channel(var/new_channel)
+	power_channel = new_channel
+
+// Sets the idle_power_usage var
+/obj/machinery/proc/update_idle_power_usage(var/new_power_usage)
+	idle_power_usage = new_power_usage
+
+// Sets the active_power_usage var
+/obj/machinery/proc/update_active_power_usage(var/new_power_usage)
+	active_power_usage = new_power_usage
+

```

Comment:
I'd advise using something like this (please double-check this, didn't actually compile it), to ensure varedit modifications work correctly. While it does nothing at this point, if you eventually want to implement static area power this ensures VV modifications call these setters instead of setting these vars directly.
```suggestion

/obj/machinery/proc/vv_edit_var(var/var_name, var/new_value)
	if(var_name == "power_channel")
		update_power_channel(new_value)
		return TRUE
	else if (var_name == "idle_power_usage")
		update_idle_power_usage(new_value)
		return TRUE
	else if (var_name == "active_power_usage")
		update_active_power_usage(new_value)
		return TRUE
	return ..()
```

Reply:
That is a good idea! I'll add it here.